pytorch_lightning/profiler/profilers.py

```diff
@@ -469,14 +454,19 @@ def _create_profiler(self, action_name, profiler, enter=True):
             pr = pr.__enter__()
         self.profiler = pr
 
-    def _stop(self, action_name: str) -> None:
+    @property
+    def function_events(self):
+        return self.profiler.function_events
+
+    def _stop(self, action_name: str, from_stop: bool = False) -> None:
```

Comment:
what is `from_stop`?

Reply:
Renamed to `triggered_by_stop_function`.